metadata-ingestion/tests/integration/vertica/docker-compose.yml

```diff
@@ -1,25 +1,22 @@
-version: '3.9'
+version: "3.3"
```

Comment:
why was this changed?

Reply:
Docker compose was giving an version error for vertica while running the tests .